docs/quickstart.rst

```diff
@@ -103,7 +103,7 @@ Make a call
     })
 
 Docs:
-`https://docs.nexmo.com/voice/voice-api/api-reference#call\_create <https://docs.nexmo.com/voice/voice-api/api-reference#call_create?utm_source=DEV_REL&utm_medium=github&utm_campaign=python-client-library>`__
+`https://docs.vonage.com/voice/voice-api/api-reference#call\_create <https://docs.vonage.com/voice/voice-api/api-reference#call_create?utm_source=DEV_REL&utm_medium=github&utm_campaign=python-client-library>`__
```

Comment:
This needs correct to point to the developer docs, this URL isn't valid. `docs.nexmo.com` is also just a redirect, so this should point to the current URL.